### PR TITLE
[WIP] Add a model evaluation script.

### PR DESCRIPTION
**Pull request by luomai**

### CODE REVIEW

**Comment by mandeman** [comment]
args.model doesn't exist. Throws an error.

**Comment by luomai** [comment]
@mandeman Yes, the script is still under development. We mark it [WIP] for this purpose.

File: eval.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        body_part = human.body_parts[coco_id]
+        keypoints.extend([round_int(body_part.x * image_w), round_int(body_part.y * image_h), 2])
+    return keypoints
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Tensorflow Openpose Inference')
+    # parser.add_argument('--resize', type=str, default='0x0', help='if provided, resize images before they are processed. default=0x0, Recommends : 432x368 or 656x368 or 1312x736 ')
+    # parser.add_argument('--resize-out-ratio', type=float, default=8.0, help='if provided, resize heatmaps before they are post-processed. default=8.0')
+    parser.add_argument('--base-model', type=str, default='vgg', help='vgg | mobilenet')
```
Comment (zsdonghao) [inline eval.py]: follow config.py is clear

**Comment by bela127** [comment]
Hi,

I did proposed #184
But if your already working on it, I guess I don't have to implement it all by myself...

Can I help you with anything?

**Comment by bela127** [comment]
all tested and fixed/edited some things #184, on my system it is working now...

if code is wanted: https://github.com/bela127/openpose-plus

**Comment by luomai** [comment]
Hi @bela127 the developer working on this PR has worked on other projects else. It would be great if you can contribute to this PR. Many thanks!

**Comment by bela127** [comment]
@luonai

Always willing to help on good projects, what are Todo s one should work on?

**Comment by bela127** [comment]
The PlotHumans function will fail if a lot of images are processed...
Because of a Python recursion stack of overflow...
It's because of the inner workings of mathplot...
In the Funktion plot.close should be called to solve the problem...

Alternative the stack limit must be extended (not a good alternative)

**Pull request closed without merging**